packages/cli/test/ipfs/ipfs-view.test.js

```diff
@@ -0,0 +1,13 @@
+import test from 'ava'
+import parseCli from '../parseCli'
+
+test.serial('ipfs view readme directory', async t => {
+  const stdout = await parseCli([
+    'ipfs',
+    'view',
+    'QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG',
+    '--debug',
+  ])
+
+  t.assert(stdout.includes('about'), "Directory doesn't contain about file")
```

Comment:
Can we make sure all links we expect are fetched correctly?

Reply:
Indeed good idea, I added a check for the other files but skipped the test. There seems to be a bug in the command, I created an issue for it: https://github.com/aragon/aragon-cli/issues/1246